package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "solidus",
+  "version": "0.0.1",
```

Comment:
I think we should not mimic to be a npm package.

Can we remove all of the config and reduce it to the bare minimum to satisfy eslint?

Reply:
I don't know another way to be able to use an external npm package (eslint, etc..) without a package.json file. A package.json file must contain at least a name and a version. What we could do is enforcing it to be private to make things clear with the `{"private": "true"}`. source [here](https://stackoverflow.com/questions/57365415/bare-minimum-package-json)